lib/ret/api/token_utils.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Ret.Api.TokenUtils do
+  @moduledoc """
+  Utility functions for generating API access tokens.
+  """
+  alias Ret.Account
+  alias Ret.Api.{Token, Scopes}
+
+  def gen_app_token(scopes \\ [Scopes.read_rooms(), Scopes.write_rooms(), Scopes.create_accounts()]) do
+    Token.encode_and_sign(nil, %{
+      subject_type: :app,
+      scopes: scopes,
+      account_id: nil
+    })
+  end
+
+  def gen_token_for_account(%Account{} = account, scopes \\ [Scopes.read_rooms(), Scopes.write_rooms()]) do
+    Token.encode_and_sign(nil, %{
+      subject_type: :account,
+      scopes: scopes,
+      account_id: account.account_id
+    })
+  end
```

Comment:
Again, I realize this is probably just temporary, but I just wanted to flag that we should discourage tokens that have all scopes enabled by default. Best practice is that tokens should have no scopes by default, and API users should have to explicitly opt-in to the scopes they need.

Reply:
Yep -- I'll address this once there's an interface/API for actually creating the credentials. (I'm doing it on the command line or in an iex session now).